### PR TITLE
Eslint

### PR DESCRIPTION
**Pull request by akoushke**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,73 @@
+{
+  "parser": "babel-eslint",
+  "extends": [
+    "@webex/eslint-config-react",
```
Comment (lalli-flores) [inline .eslintrc]: Should we extend from Airbnb's instead of this one?

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    "plugin:jsx-a11y/recommended",
+    "plugin:prettier/recommended",
+    "prettier/react"
+  ],
+  "plugins": ["react", "jsx-a11y", "import", "prettier", "react-hooks"],
```
Comment (lalli-flores) [inline .eslintrc]: Should we add the JSDoc plugin?

File: .eslintrc (hunk 1 above)
Comment (akoushke) [reply .eslintrc]: can we do that after we come up with our standard eslint configuration

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        "array": true,
+        "object": true
+      },
+      {
+        "enforceForRenamedProperties": false
+      }
+    ],
+    "react-hooks/rules-of-hooks": "error", // Checks rules of Hooks
+    "react-hooks/exhaustive-deps": "warn", // Checks effect dependencies
+    "require-jsdoc": "warn"
```
Comment (lalli-flores) [inline .eslintrc]: Can remove this since we're adding the plugin

File: .circleci/config.yml [hunk 4]
```diff
@@ -35,8 +35,19 @@ jobs:
           root: .
           paths: node_modules
 
+  test:
+      executor: main-executor
```
Comment (lalli-flores) [inline .circleci/config.yml]:
```suggestion
      executor: node-erbium-executor
```

File: .eslintrc [hunk 5]
```diff
@@ -0,0 +1,72 @@
+{
+  "parser": "babel-eslint",
+  "extends": [
+    "@webex/eslint-config-react",
+    "plugin:jsx-a11y/recommended",
+    "plugin:prettier/recommended",
+    "prettier/react"
+  ],
+  "plugins": ["import", "jsdoc", "jsx-a11y", "prettier", "react", "react-hooks"],
```
Comment (lalli-flores) [inline .eslintrc]: We should also add the `jest` plugin

Comment (akoushke) [reply .eslintrc]: sure

**Comment by lalli-flores** [comment]
Minor thing, if you can lower case the `Add` in your index commit message but approved otherwise

**Comment by adamweeks** [comment]
> Minor thing, if you can lower case the `Add` in your index commit message but approved otherwise

Not to pile on, but is there a reason `Index` is capitalized?

File: package.json [hunk 6]
```diff
@@ -4,7 +4,9 @@
   "main": "dist/webexWidgets.cjs.js",
   "module": "dist/webexWidgets.esm.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "prettier": "prettier \"src/**/!(static)/**/*.*(js|jsx)\"",
```
Comment (adamweeks) [inline package.json]: Let's not use prettier, just the eslint plugin for prettier rules.

File: .eslintrc (hunk 1 above)
Comment (adamweeks) [reply .eslintrc]: I agree with @lalli-flores, we should use the newly agreed upon eslint config instead of using the soon to be deprecated eslint-config.

**Pull request merged**